tools/protodoc/BUILD

```diff
@@ -38,3 +39,17 @@ py_binary(
         requirement("PyYAML"),
     ],
 )
+
+protodoc_rule(
+    name = "generate_api_docs",
+    deps = ["@envoy_api_canonical//:v3_protos"],
+)
+
+genrule(
+    name = "api_rst",
```

Comment:
this doesnt work - i just created it in the hope it would invoke the `actions.run` in some way